### PR TITLE
PKG-1197: Update to v0.29.1

### PR DESCRIPTION
**Pull request by pseudoyim**
Jira: https://anaconda.atlassian.net/browse/PKG-1197

### CODE REVIEW

File: recipe/meta.yaml [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {% set name = "xlwings" %}
-{% set version = "0.27.15" %}
-{% set sha256 = "9fcfbc00a2e0dcb4291c86e5816e31feefaff7e59a3c31f2c7bc73692b9469be" %}
+{% set version = "0.29.1" %}
+{% set sha256 = "88fe1050990ebe838302b4b6aafeb6e5c9620a329a50744e344288a7012360c3" %}
 
 package:
   name: {{ name|lower }}
```
Comment (skupr-anaconda) [inline recipe/meta.yaml:6]: Missing `maturin` in `host`, see https://github.com/xlwings/xlwings/blob/88e1c8c8afd148ad29157cc620e1b0f52a7b7130/pyproject.toml#L2

**Review by varlackc — COMMENTED** [review]
There is a change in the `dev_url` from:

https://github.com/ZoomerAnalytics/xlwings

to

https://github.com/xlwings/xlwings

**Review by ELundby45 — APPROVED** [review]
LGTM

File: recipe/meta.yaml (hunk 1 above)
Comment (pseudoyim) [reply recipe/meta.yaml:6]: Not adding `maturin` for now.

**Pull request merged**